docs/mastering-plone/about_mastering.md

```diff
@@ -30,17 +30,18 @@ Thank you for reading and your charity with non-native English speakers / writer
 
 ## Upcoming Trainings
 
-Plone Conference 2021, end of the year
+Next Plone Conference, end of the year.
```

Comment:
I doubt October is end of year. If at it, I would fi it too and link to ploneconf.org. It redirects to the last or next conference.
```suggestion
Next [Plone Conference](https://ploneconf.org/).
```